torntools/scripts/content/travel/ttTravelHome.js

```diff
@@ -847,12 +847,12 @@ function warnOnTimeout() {
 				travelTime = travelTimeArray[0] * 60 * 60 + travelTimeArray[3] * 60;
 			}
 			let timeoutFor;
-			if (2 * travelTime >= userdata.education_timeleft && 2 * travelTime >= userdata.city_bank.time_left) {
-				timeoutFor = "education course and bank investment";
-			} else if (2 * travelTime >= userdata.education_timeleft) {
+			if (userdata.education_timeleft && 2 * travelTime >= userdata.education_timeleft) {
 				timeoutFor = "education course";
-			} else if (2 * travelTime >= userdata.city_bank.time_left) {
+			} else if (userdata.city_bank.time_left && 2 * travelTime >= userdata.city_bank.time_left) {
 				timeoutFor = "bank investment";
+			} else if (userdata.education_timeleft && userdata.city_bank.time_left && 2 * travelTime >= userdata.education_timeleft && 2 * travelTime >= userdata.city_bank.time_left) {
```

Comment:
This won't ever trigger?

Reply:
It will.
When you have both times left and both times are greater than travel time, it will trigger.

Reply:
I just checked, and it won't.
When both education and bank are done, it will only show "education", since that check passes first and it ignored the others.

Reply:
Oh it should be on top. Yes. Sorry.